tests/unit/s2n_sike_p503_kat_test.c

```diff
@@ -0,0 +1,100 @@
+/*
+ * Copyright 2018 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License").
+ * You may not use this file except in compliance with the License.
+ * A copy of the License is located at
+ *
+ *  http://aws.amazon.com/apache2.0
+ *
+ * or in the "license" file accompanying this file. This file is distributed
+ * on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either
+ * express or implied. See the License for the specific language governing
+ * permissions and limitations under the License.
+ *
+ * Modified from PQCgenKAT_kem.c
+ * Created by Bassham, Lawrence E (Fed) on 8/29/17.
+ * Copyright © 2017 Bassham, Lawrence E (Fed). All rights reserved.
+ */
+
+#include "s2n_test.h"
+#include "s2n_nist_kats.h"
+#include "pq-crypto/sike/sike_p503_kem.h"
+#include "pq-crypto/pq-random.h"
+
+#define RSP_FILE_NAME "kats/sike_p503.kat"
+
+int main(int argc, char **argv, char **envp) {
+    BEGIN_TEST();
```

Comment:
Does SIKE work with the FIPS OpenSSL already? Should this skip the tests like BIKE? Or since we've accepted modifying the code should we apply your BIKE path that makes it work the correct FIPS OpenSSL API?

Reply:
To the best of my knowledge SIKE does not depends on OpenSSL at all (FIPS or not). 
I will apply BIKE patch to a future pull request.